scanning/disa-alignment/shared.py

```diff
@@ -1,12 +1,47 @@
-import re
+from collections import namedtuple
+import xml.etree.ElementTree as ET
 
 from lib import results
 
+nsmap = {
+    "xccdf": "http://checklists.nist.gov/xccdf/1.2",
+}
+STIG_REF = "https://public.cyber.mil/stigs/srg-stig-tools/"
+CCE = "https://ncp.nist.gov/cce"
+SSG_RULE_PREFIX = "xccdf_org.ssgproject.content_rule_"
+DISA_RULE_PREFIX = "xccdf_mil.disa.stig_rule_"
+
+DISARuleResult = namedtuple('DISARuleResult', ['rule_id', 'result'])
 
 profile = 'stig'
 
 shared_cmd = ['oscap', 'xccdf', 'eval', '--progress']
 
+class SSGRuleResult:
+    def __init__(self, rule_id, cce_id, rule_title, stig_ids, result):
+        self.rule_id = rule_id
+        self.cce_id = cce_id
+        self.rule_title = rule_title
+        self.stig_ids = stig_ids
+        self.result = result
+        self.stig_ids_results = {}
+        self.final_result = None
+
+    def __repr__(self):
+        return (
+            f"SSGRuleResult(rule_id='{self.rule_id}', cce_id='{self.cce_id}',"
+            f"rule_title='{self.rule_title}', stig_ids={self.stig_ids}', "
+            f"result='{self.result}', "
+            f"comparison_with_stig_ids={self.stig_ids_results}, "
+            f"final_result={self.final_result})"
+        )
+
+
+class ComparisonResult:
+    SAME = "SAME"
+    DIFFERENT = "DIFFERENT"
+    MISSING = "MISSING"
```

Comment:
Unless you print the `"SAME"`, `"DIFFERENT"`, etc. strings somewhere, using `enum` is cleaner and faster.

Do
```python
import enum
```
at the top, then

```suggestion
class ComparisonResult(enum.Enum):
    SAME = enum.auto()
    DIFFERENT = enum.auto()
    MISSING = enum.auto()
```
See https://docs.python.org/3/library/enum.html .